idd/Energy+.idd.in

```diff
@@ -104732,7 +104865,7 @@ Output:EnvironmentalImpactFactors,
 EnvironmentalImpactFactors,
    \memo Used to help convert district and ideal energy use to a fuel type and provide total carbon equivalent with coefficients
    \memo Also used in Source=>Site conversions.
-  N1, \field District Heating Efficiency
+  N1, \field District Heating Water Efficiency
```

Comment:
Should `N3, \field Steam Conversion Efficiency` change to `District Heating Steam Conversion Efficiency`?

Reply:
@mjwitte Thank you for catching this. I'll fix it.